src/http/http_rpc_context.h

```diff
@@ -339,4 +381,92 @@ namespace enclave
       packed,
       raw_pbft);
   }
+
+  inline std::shared_ptr<enclave::RpcContext> make_fwd_rpc_context(
+    std::shared_ptr<enclave::SessionContext> s,
+    const std::vector<uint8_t>& packed,
+    enclave::FrameFormat frame_format,
+    const std::vector<uint8_t>& raw_pbft = {})
+  {
+    http::SimpleRequestProcessor processor;
+
+    LOG_DEBUG_FMT("PARSING FORWARDED REQUEST");
+    LOG_DEBUG_FMT("{:02x}", fmt::join(packed, " "));
+    LOG_DEBUG_FMT("{}", std::string(packed.begin(), packed.end()));
```

Comment:
Is this staying? And if so, should it be `LOG_TRACE_FMT`?

Reply:
No, that's not useful anymore.